### ISSUE TITLE
Add VAE to test-suits

### ISSUE THREAD

**Issue by whyhardt** [AutoResearch/EEG-GAN#100]
Most test-suits incorporate GAN and AE-GAN. Would be nice to cover VAE as well.

**Comment by chadcwilliams** [comment]
Agreed! I can add that in. I wasn't sure if we were going to keep VAE for the actual pip release - something to discuss in our next meet.
